src/RFC-0121_ConsensusEncoding.md

```diff
@@ -343,12 +343,12 @@ pub struct TransactionKernel {
 
 The transaction kernel is encoded as follows:
 
-- `input_version` - 1 byte
-- `features` - [OutputFeatures]
-- `fee` - [RistrettoPublicKey]
-- `lock_height` - [TariScript Stack]
-- `excess` - [Commitment]
-- `excess_sig` - [Signature]
+1. `input_version` - [varint] 1 byte
```

Comment:
This isn't varint, just a single byte since we only use a u8 for input/output versions

https://github.com/tari-project/tari/blob/9162b4fa47cdc31e0cbbdfab0b8459eb5c58f7c5/base_layer/core/src/transactions/transaction_components/transaction_input_version.rs#L46

Reply:
Oh it's the kernel, this is named `version` in the code
```suggestion
1. `version` - 1 byte
```